api/v1alpha1/enterprisecontractpolicy_types.go

```diff
@@ -28,9 +28,9 @@ type EnterpriseContractPolicySpec struct {
 	// Description of the policy or it's intended use
 	// +optional
 	Description string `json:"description,omitempty"`
-	// One or more source urls for the policy rules
+	// One or more groups of policy rules
 	// +kubebuilder:validation:MinItems:=1
-	Sources []string `json:"sources"`
+	Sources []Source `json:"source,omitempty"`
```

Comment:
Since it's a list, I was thinking maybe "sources" is a more intuitive key name vs "source". Is there some reasoning behind making a change to "source"?

Reply:
Typo. Thanks for catching!